deployment/src/main/java/io/quarkiverse/wiremock/devservice/WireMockServerProcessor.java

```diff
@@ -69,10 +80,19 @@ DevServicesResultBuildItem setup(LaunchModeBuildItem launchMode, LiveReloadBuild
 
     @BuildStep(onlyIf = { WireMockServerEnabled.class, GlobalDevServicesConfig.Enabled.class })
     @Consume(DevServicesResultBuildItem.class)
-    DevServiceDescriptionBuildItem renderDevServiceDevUICard(WireMockServerBuildTimeConfig config) {
+    DevServiceDescriptionBuildItem renderDevServiceDevUICard() {
         return new DevServiceDescriptionBuildItem(DEV_SERVICE_NAME, null, devService.getConfig());
     }
 
+    @BuildStep(onlyIf = { WireMockServerEnabled.class, GlobalDevServicesConfig.Enabled.class, IsDevelopment.class })
+    void watchWireMockConfigFiles(WireMockServerBuildTimeConfig config,
+            BuildProducer<HotDeploymentWatchedFileBuildItem> items) {
+        listFiles(Paths.get(config.filesMapping(), MAPPINGS), Paths.get(config.filesMapping(), FILES)).forEach(file -> {
```

Comment:
According to https://quarkus.io/guides/all-builditems , [HotDeploymentWatchedFileBuildItem](https://github.com/quarkusio/quarkus/blob/main/core/deployment/src/main/java/io/quarkus/deployment/builditem/HotDeploymentWatchedFileBuildItem.java) actually supports passing a Glob pattern, hence you could probably instruct it to watch all files under the Mappings and Files directories (including when new ones added?)

Reply:
I've analyzed the `glob-pattern` feature back in the days. However, it does not convince me since the API is not very intuitive. Furthermore, it also doesn't work for new files (please see https://github.com/quarkusio/quarkus/issues/25338). Anyway, the current implementation already watches all files under the Mappings and Files directories.